src/components/Slider/index.js

```diff
@@ -0,0 +1,106 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { uniqueId } from './../../libs/utils';
+import Label from './label';
+import './style.css';
+import RenderIf from '../RenderIf';
+
+export default class Slider extends Component {
+    constructor(props) {
+        super(props);
+        this.sliderId = uniqueId('slider-id');
+        this.errorMessageId = uniqueId('error-message');
+    }
+
+    getContainerClassName() {
+        const { className } = this.props;
+        return classnames('rainbow-slider_container', className);
+    }
+
+    render() {
+        const {
+            label,
+            value,
+            min,
+            max,
+            step,
+            error,
+            disabled,
+            onBlur,
+            onChange,
+            onClick,
+            onFocus,
+            style } = this.props;
+        return (
+            <div className={this.getContainerClassName()} style={style}>
+                <Label label={label} sliderId={this.sliderId} />
+                <div className="rainbow-slider">
+                    <input
+                        id={this.sliderId}
+                        className="rainbow-slider_range"
+                        type="range"
+                        value={value}
+                        min={min}
+                        max={max}
+                        step={step}
+                        disabled={disabled}
```

Comment:
here you forgot to pass the aria-describedby that is associated with the error id, but remember that only is passed if there is an error otherwise is undefined.